admin-dev/themes/new-theme/js/components/grid.js

```diff
@@ -32,10 +32,12 @@ export default class Grid {
   /**
    * Grid's selector
    *
-   * @param gridSelector
+   * @param {string} gridPanelSelector
    */
-  constructor(gridSelector) {
-    this.$grid = $(gridSelector);
+  constructor(gridPanelSelector) {
+    this.$gridPanel = $(gridPanelSelector);
+    this.gridId = this.$gridPanel.data('grid-id');
+    this.$grid = this.$gridPanel.find('#' + this.gridId + '_grid');
```

Comment:
`this.$gridPanel.find` is useless, it's an ID, must be uniq on the page so you can direct use `$('#');`